contexts/tripContext.tsx

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import React, { useContext, useEffect, useState } from 'react';
+import { Alert } from 'react-native';
+import { useUserContext } from './userContext';
+
+const TripContext = React.createContext<any>({
+  id: null,
+  start: '',
+  end: '',
+  destination: '',
+  longitude: '',
+  latitude: '',
+  Events: [],
+  Users: [],
+  Flights: [],
+  Hotels: [],
+});
+
+export function useTripContext() {
+  return useContext(TripContext);
+}
```

Comment:
I love that you're using the same format I did. I can just come in and use this without knowing the implementation details now.

+1 for separation of concerns.

Reply:
nice